uuid.go

```diff
@@ -305,6 +312,33 @@ func (u *UUID) Scan(src interface{}) error {
 	return fmt.Errorf("uuid: cannot convert %T to UUID", src)
 }
 
+// Value implements the driver.Valuer interface.
+func (u NullUUID) Value() (driver.Value, error) {
+	if !u.Valid {
+		return nil, nil
+	}
+	// Delegate to UUID Value function
+	return u.UUID.Value()
+}
+
+// Scan implements the sql.Scanner interface.
+func (u *NullUUID) Scan(src interface{}) error {
+	if src == nil {
+		*u = NullUUID{Nil, false}
+		return nil
+	}
+
+	// Delegate to UUID Scan function
+	err := u.UUID.Scan(src)
+	if err != nil {
+		u.Valid = false
```

Comment:
@cbandy Sorry, I overlooked your note. I think we can directly return the value of `UUID.Scan` as you suggested. I initially thought it would be better to invalid the object in case of error, but as you said `u` is meaningless in case of error.